docs/product-development-guide/src/main/asciidoc/assembly_security-management.adoc

```diff
@@ -0,0 +1,66 @@
+[id='assembly_security-management']
+= Security Management
+
+{PRODUCT} {RELEASE} introduces enhanced security features for Business Central that enable administrators to specify the home perspective for users, as well as control access to assets. For a detailed description of these new features, see the "Business Central Settings" section of the _{PRODUCT} User Guide_. 
+
```

Comment:
add a comma before which

Reply:
I'Which' in this case is used to introduce a restrictive clause. To remove the clause would change the meaning of the sentence, therefore a comma is not used (only for restrictive clauses).